doc/api/http2.md

```diff
@@ -34,34 +45,34 @@ server.on('stream', (stream, headers) => {
   stream.end('<h1>Hello World</h1>');
 });
 
-server.listen(80);
+server.listen(8443);
 ```
 
-Note that the above example is an HTTP/2 server that does not support SSL.
-This is significant as most browsers support HTTP/2 only with SSL.
-To make the above server be able to serve content to browsers,
-replace `http2.createServer()` with
-`http2.createSecureServer({key: /* your SSL key */, cert: /* your SSL cert */})`.
+### Client-side example
 
 The following illustrates an HTTP/2 client:
 
 ```js
-const http2 = require('http2');
-
-const client = http2.connect('http://localhost:80');
+var http2 = require('http2');
+const client = http2.connect('https://localhost');
```

Comment:
Since the default example is using a `createSecureServer`, maybe this could mention that one might need either `rejectUnauthorized: false` or to pass the `ca` in order for self-signed certs to work.

Also, it should probably connect on port 8443 to match the example above.